ix-dev/community/versitygw/item.yaml

```diff
@@ -0,0 +1,9 @@
+categories:
+- storage
+icon_url: "TODO"
```

Comment:
I don't have an icon for this, so I left this as "TODO"; let me know if you want me to do something else.